integreat_cms/cms/templates/organizations/organization_form.html

```diff
@@ -15,9 +19,25 @@ <h1 class="heading">
 				{% trans 'Create new organization' %}
 			{% endif %}
 		</h1>
-		{% if perms.cms.change_organization %}
-			<button class="btn">{% trans 'Save' %}</button>
-		{% endif %}
+		<div class="flex flex-wrap justify-between gap-4"> 
+			{% if form.instance.id and perms.cms.delete_organization %}
+				<div class="flex flex-wrap gap-4">
+					<button title="{% trans 'Delete organization' %}" class="btn confirmation-button btn-red"
+							data-confirmation-title="{{ delete_dialog_title }}"
+							data-confirmation-text="{{ delete_dialog_text }}"
+							data-confirmation-subject="{{ form.name.value }}"
+							data-action="{% url 'delete_organization' region_slug=request.region.slug slug=form.instance.slug %}">
+						<i data-feather="trash-2" class="mr-2"></i>
```

Comment:
```suggestion
						<i icon-name="trash-2" class="mr-2"></i>
```
Ah, and we currently changed our icon dependency

Reply:
Alright. Good to know for the future :D